README.md

```diff
@@ -100,6 +100,11 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Things To Watch Out For While Working Remotely](https://www.toptal.com/remote/remote-work-burnout-a-cautionary-tale)
   1. [How to Work Remotely and Still Be the Best](https://www.toptal.com/freelance/how-to-work-remotely-and-still-be-the-best)
   1. [The Ultimate Remote Team Culture](https://www.toptal.com/remote/the-ultimate-remote-culture)
+  1. [How to Handle Client Work Remotely: Our Communication Stack](https://marsbased.com/blog/2015/12/07/how-to-handle-client-work-remotely-our-communication-stack/)
+  1. [Our Six Tips for Working Remotely](https://marsbased.com/blog/2015/06/15/six-tips-for-remote-working/)
+  1. [Dealing With Your First Hire in a Distributed Team](https://marsbased.com/blog/2014/10/25/dealing-with-your-first-hire-in-a-distributed-team/)

```

Comment:
I'm not sure this is entirely relevant to remote working; it covers only a particular thing, and there's no follow up later. I'd rather this wasn't in this PR.

Reply:
Done!